src/documentation/code-review.md

```diff
@@ -16,48 +16,48 @@ An objective is, therefore, to *keep the effort for code review proportional* to
+In research projects like Hexatomic, we propose that this should be the general responsibility of the maintainer(s).
 They can delegate the code reviews to others if appropriate or necessary.
-E.g., if the maintainer is submitting a pull request by themselves, another person should responsible for code review.
-Delegating code reviews can be helpful to distribute the workload between several persons.
+E.g., if the maintainer is submitting a pull request themself, another person should be responsible for code review.
+Delegating code reviews can help distribute the workload over several people.
 
 In Hexatomic we have three areas where contributions require code review.
 
 ### Project documentation
 
-The project documentation is similar to a formal project report.
-We, therefore, argue that ideally, the documentation should be reviewed by the Project Instructors (PIs).
-To avoid a “chicken or egg” problem, project maintainers (who are also the researches in the project) can review each other pull requests to bootstrap the initial documentation, e.g. about how code review is performed.
+One of the purposes of the project documentation is to report progress and outcomes of our research, similar to a formal project report. In fact, it is our approach to generate the project report based on the project documentation.
+We therefore argue that ideally, the documentation should be reviewed by the principal investigators (PIs).
+To avoid a "chicken and egg" problem, project maintainers who are also the researches in the project. can review each other's pull requests to bootstrap the initial documentation, e.g., about how code review is performed.
 
-When the project is finished, the project documentation will probably not need any updates.
+Once the project is completed, the project documentation will probably not need any updates.
 If an update is still necessary, the maintainers can decide who should review it.
 
 ### User, developer and maintainer documentation
 
-The maintainer should suggest a suitable person for a code review of documentation.
-This can include other project members and PIs who are not developers but also external users.
+The maintainer should suggest a suitable person for a review of changes to documentation.
+This can include other project members and PIs who are not developers, but also external users.
 We argue that especially user documentation can benefit from a user perspective in the review process.
-This means that the documentation updates and the review interface must be usable for informed users.
+This means that informed users must be enabled to make updates in and add changes to the documentation, and to use the review user interface. In order to achieve this, the code review process and involved technology is documented in the maintainer documentation, to share with reviewers.
```

Comment:
Do we track this requirement for the maintainer docs? The process and tech documentation involved in code reviews? If not, can you create an issue for this in the `hexatomic` repo please?

Reply:
I think this is a mixture of
- existing maintainer/dev documentation
- <https://help.github.com/en/articles/reviewing-proposed-changes-in-a-pull-request>
- #13

Reply:
Updated #13 to reflect this.